### PR TITLE
Improvements to app documentation

### PR DESCRIPTION
**Pull request by mikewilburn**
This PR:
- renames `docs/index.md` to `docs/README.md`
- adds table of contents to (2) README files. The TOC was generated using MDTOC.
- links to `docs/README.md` from root README.md
- corrects minor structure and formatting changes in root README.md (85b2d06)
- describes patch changes in RELEASE.md
- increments app version patch number

@nCastle1 I'm less familiar with versioning in this project, so let me know if I missed any files in f76a7a5 where I should have also incremented the app version.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -4,6 +4,32 @@ This doc describes Indoor Routing, including how you can customize it to work fo
 
 > **Looking for ArcGIS Indoors?** ArcGIS Indoors is Esri's solution for indoor mapping, location tracking, and wayfinding; no code required. [Learn more about ArcGIS Indoors](https://www.esri.com/en-us/arcgis/products/arcgis-indoors).
```
Comment (esreli) [inline docs/README.md:5]: Maybe this would better serve the document if it was placed above the top h1? Just a suggestion...

File: RELEASE.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Release notes
 
+## 2.0.2
```
Comment (esreli) [inline RELEASE.md:3]: It appears the convention in this doc is to precede the version number with 'Release'. 
```suggestion
## Release 2.0.2
```

File: README.md [hunk 3]
```diff
@@ -13,14 +34,18 @@ This repo provides an example app called [Indoor Routing for iOS](https://develo
...
+
 ## Development Instructions
 This Indoor Routing repo is a Xamarin Studio Project that can be directly cloned and imported into Xamarin Studio or Visual Studio.
 
 ### Fork the repo
 **Fork** the [Indoor Routing Xamarin](https://github.com/Esri/indoor-routing-xamarin/fork) repo.
 
 ### Clone the repo
-Once you have forked the repo, you can make a clone
+Once you have forked the repo, you can make a clone.
```
Comment (esreli) [inline README.md:48]: 👍

File: README.md [hunk 4]
```diff
@@ -35,10 +60,10 @@ If you make changes in the fork and would like to [sync](https://help.github.com
 
 If there are changes made in the Original repository, you can sync the fork to keep it updated with upstream repository.
 
-1. In the terminal, change the current working directory to your local project
-2. Type ```git fetch upstream``` to fetch the commits from the upstream repository
+1. In the terminal, change the current working directory to your local project.
+2. Type ```git fetch upstream``` to fetch the commits from the upstream repository.
 3. ```git checkout master``` to checkout your fork's local master branch.
-4. ```git merge upstream/master``` to sync your local `master' branch with `upstream/master`. **Note**: Your local changes will be retained and your fork's master branch will be in sync with the upstream repository.
+4. ```git merge upstream/master``` to sync your local `master` branch with `upstream/master`. **Note**: Your local changes will be retained and your fork's master branch will be in sync with the upstream repository.
```
Comment (esreli) [inline README.md:66]: 👍

**Comment by mikewilburn** [comment]
@esreli I think your suggestion to float the ArcGIS Indoors message to the top was a good one. I also made that other change you noticed - good catch (all these RELEASE.md updates have me mixing formats). Let me know if things now look good from your perspective.

**Pull request merged**